benchmarks/nemo/nemolite2d/psykal/psyclone_scripts/ocl_transform.py

```diff
@@ -2,31 +2,83 @@
 function via the -s option. Applies OpenCL to the Schedule so
 that PSyclone will generate an OpenCL PSy layer. '''
 
+import os
 from psyclone.psyGen import TransInfo
 from psyclone.domain.gocean.transformations import \
     GOMoveIterationBoundariesInsideKernelTrans
+from psyclone.configuration import Config
+
+FUCTIONAL_PARALLELISM = True
```

Comment:
Please add comments describing each of these settings.

Reply:
Good idea, added comments.

Reply:
Great, thanks.